pkg/storage/stores/indexshipper/uploads/index_set.go

```diff
@@ -65,13 +65,9 @@ func (t *indexSet) Add(idx index.Index) {
 	t.index[idx.Name()] = idx
 }
 
-func (t *indexSet) ForEach(_ context.Context, doneChan <-chan struct{}, callback index.ForEachIndexCallback) error {
+func (t *indexSet) ForEach(callback index.ForEachIndexCallback) error {
```

Comment:
do you know the story behind that unused context? I see we're removing it, but I'm curious why it was ever there in the first place?